### PR TITLE
Sad Trombone implant is now buildable!

### PR DESCRIPTION
**Pull request by GunHog**
:cl: Gun Hog
add: In an effort to reduce the demoralizing effects that accompany the death of a crew member, Nanotrasen has provided the prototype designs for the Sad Trombone implant. It can be fabricated at your station's protolathe, if you manage to secure the ever elusive Bananium ore.
/:cl:

You may now construct the Sad Trombone implant in Science. Easy tech requirements, but requires Bananium (because it is a clown item).

### CODE REVIEW

**Comment by Miauw62** [comment]
tgstation is literally a meme

**Comment by lordpidey** [comment]
What happens if someone has multiple of these implants?

Also, are they deconstructable?

**Comment by GunHog** [comment]
>What happens if someone has multiple of these implants?

Also, are they deconstructable?

I have not tried multiple implants. It would be funny if it worked, though. And yes, they can be deconstructed in the DA, but yield no useful tech. By the time the Clown finds Bananium and brings it back, you should have Science levels well beyond the requirements.

**Comment by KorPhaeron** [comment]
you can't implant multiple

File: code/modules/research/designs/medical_designs.dm [hunk 1]
```diff
@@ -287,6 +287,16 @@
 	build_path = /obj/item/weapon/implantcase
 	category = list("Medical Designs")
 
+/datum/design/implantcase
```
Comment (Supermichael777) [inline code/modules/research/designs/medical_designs.dm]: COPY PASTA DETECTED

**Pull request merged**